syntax = "proto3";
package nanoswap.message;

// This schema is used to find the credit identity for someone based on their goverment issued ID

// ipfs file title: <id field type>.<id field content>, ex: "0.123-45-6789" (since 0 is the enum value of UNITED_STATES_AMERICA_FEDERAL_SOCIAL_SECURITY_NUMBER)
message Lookup {
    string credit_identity = 1; // uuid of the user
}
